state/migration_export.go

```diff
@@ -202,6 +204,17 @@ func (e *exporter) readBlocks() (map[string]string, error) {
 	return result, nil
 }
 
+func (e *exporter) modelStatus() error {
+	statusArgs, err := e.statusArgs(modelGlobalKey)
+	if err != nil {
+		return errors.Annotatef(err, "status for model")
+	}
+
+	e.model.SetStatus(statusArgs)
```

Comment:
Well, we do have dump-model command which does an export.

Right now, we only use exports for migration, but that doesn't mean this won't change in the future.